arcgis-ios-sdk-samples/Content Display Logic/Controllers/SampleInfoViewController.swift

```diff
@@ -24,14 +24,8 @@ class SampleInfoViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        // We must construct the web view in code as long as we support iOS 10.
-        // Prior to iOS 11, there was a bug in WKWebView.init(coder:) that
-        // caused a crash.
-        let webView = WKWebView(frame: view.bounds)
         webView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
         webView.navigationDelegate = self
```

Comment:
Since we already dropped support for below iOS 12, there's no need to do this. I've added a WKWebView to main.storyboard and connected with an IBOutlet.